lib/krikri/ldp/rdf_source.rb

```diff
@@ -0,0 +1,29 @@
+module Krikri::LDP
+  ##
+  # Adds simple LDP persistence to ActiveTriples::Resource classes
+  # @see ActiveTriples::Resource
+  module RdfSource
+    extend ActiveSupport::Concern
+    include Krikri::LDP::Resource
+
+    def save(*)
+      result = super(dump(:ttl))
+      get({}, true)
+      result
+    end
+
+    def get(*args)
+      result = super
+      reload_ldp
+      result
+    end
```

Comment:
It would be super awesome if you could include a comment saying what object type a call to `get` will return (including the possibility of errors being thrown).